src/cbc_sdk/platform/alerts.py

```diff
@@ -1385,3 +1404,243 @@ def set_vendor_names(self, names):
             raise ApiError("One or more invalid vendor name values")
         self._update_criteria("vendor_name", names)
         return self
+
+
+class ContainerRuntimeAlertSearchQuery(BaseAlertSearchQuery):
+    """Represents a query that is used to locate ContainerRuntimeAlert objects."""
+
+    def __init__(self, doc_class, cb):
+        """
+        Initialize the ContainerRuntimeAlertSearchQuery.
+
+        Args:
+            doc_class (class): The model class that will be returned by this query.
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+        """
+        super().__init__(doc_class, cb)
+        self._bulkupdate_url = "/appservices/v6/orgs/{0}/alerts/cbanalytics/containerruntime/_criteria"
```

Comment:
Is this suppose to have `cbanalytics` in the path?

Reply:
Good catch. No.